packages/broker/src/plugins/storage/StorageConfig.ts

```diff
@@ -29,36 +29,45 @@ const getKeyFromStream = (streamId: string, streamPartition: number): StreamKey
     return `${streamId}::${streamPartition}`
 }
 
-const getKeysFromStream = (streamId: string, partitions: number) => {
-    const keys = new Set<StreamKey>()
+const getKeysFromStream = (streamId: string, partitions: number): StreamKey[] => {
+    const keys: StreamKey[] = []
     for (let i = 0; i < partitions; i++) {
-        keys.add(getKeyFromStream(streamId, i))
+        keys.push(getKeyFromStream(streamId, i))
     }
     return keys
 }
 
+const belongsToMeInCluster = (key: StreamKey, clusterSize: number, myIndexInCluster: number) => {
```

Comment:
This could be a method of `StorageConfig` class as it needs access to `clusterSize` and `myIndexInCluster`